charts/velero/README.md

```diff
@@ -50,9 +53,10 @@ helm install vmware-tanzu/velero \
 --set initContainers[0].image=velero/velero-plugin-for-<PROVIDER NAME>:<PROVIDER PLUGIN TAG> \
 --set initContainers[0].volumeMounts[0].mountPath=/target \
 --set initContainers[0].volumeMounts[0].name=plugins \
---generate-name
 ```
 
+Users of zsh might need to put quotes around key/value pairs.
```

Comment:
I never seen users had reported this issue before.
Besides, could we make sure it always need to add quotes? So we could remove the "might" term.

Reply:
When I use zsh, I didn't need quotes for every key/value pair. I only needed quotes for the ones that indexed an array. For example, these work:

```
[...clipped...]
 --set configuration.volumeSnapshotLocation.config.region=us-east-2 \
 --set "initContainers[0].name=velero-plugin-for-aws" \
[...clipped...]
```

Bash requires no quotes at all.

Reply:
I use fish shell and did not need any quotes.
Thanks for your explanation.

Reply:
BTW, which helm version you are using?

Reply:
There are a lot of MacBook users who use zsh because it is the default shell for OSX Catalina and newer. I wonder if I'm the only zsh user with the quotes problem.

Here are the versions I'm using:
```
❯ helm version
version.BuildInfo{Version:"v3.5.1", GitCommit:"32c22239423b3b4ba6706d450bd044baffdcf9e6", GitTreeState:"dirty", GoVersion:"go1.15.7"}
❯ zsh --version
zsh 5.7.1 (x86_64-apple-darwin19.0)
```

When I don't use quotes, I get this error:
```
zsh: no matches found: initContainers[0].name=velero-plugin-for-aws
```

Reply:
You're right, it's `zsh` behavior https://stackoverflow.com/a/65687265/4656380.

Reply:
I'd prefer to adding this to note _or_ add it under both helm2 and helm3.

Reply:
I'll be sure to have a note in both the Helm 2 and Helm 3 sections.